muskathlon/templates/muskathlon_my_events.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+    id="my_events_muskathlon"
+    inherit_id="website_event_compassion.my_event_details"
```

Comment:
Why not implementing this change directly in the parent template ? It will also benefit other event types than Muskathlon, and since it's our own module, we can do it in the parent.